packages/patternfly-4/react-core/src/components/Select/Select.tsx

```diff
@@ -120,6 +120,12 @@ export class Select extends React.Component<SelectProps, SelectState> {
         typeaheadFilteredChildren: React.Children.toArray(this.props.children)
       });
     }
+
+    if (prevProps.selections !== this.props.selections) {
```

Comment:
Change to `if (prevProps.selections !== this.props.selections && this.state.typeaheadActiveChild) {`